### DISCUSSION TITLE
Reaching out for mentors for project ,2. Inference of Neural model in Node.js environment using OpenVINO

### DISCUSSION THREAD

**Discussion by BlossomBreeze25** [openvinotoolkit/openvino#23206] [Google Summer of Code]
Respected sir / ma'am,
I am Ragini Devi Thakkar , 4th Semester BCA(AI) student, want to join your organization. i have knowledge about AI and languages like python , java . html , css . i am interested in this project (2. Inference of Neural model in Node.js environment using OpenVINO)and I would love to contribute to your organization but could you please tell me how to get started?

Hoping to hear from you soon.

Regards,

Ragini Devi Thakkar.

**Comment by adrianboguszewski** [comment]
@vishniakov-nikolai @almilosz please look at it

**Comment by vishniakov-nikolai** [comment]
Hello, @Ragini252000! Thank you for expressing your interest in implementing **Inference of Neural model in Node.js environment using OpenVINO**!
 
Firstly, we kindly request all participants to review the [Prerequisite task](https://github.com/openvinotoolkit/openvino/wiki/Google-Summer-Of-Code#prerequisite-task).
To begin, you may select one of the tasks listed under "good first issues" on [this page](https://github.com/orgs/openvinotoolkit/projects/3).
If you are unsure which task to choose, please feel free to send your question to us, and we will be happy to provide additional information.

I've also seen your message regarding the task: **Desktop Chat-Bot application**. As each participant can only undertake one task for implementation, you'll need to decide which one you'd like to pursue.
